packages/opentelemetry-plugin-grpc/src/grpc.ts

```diff
@@ -320,17 +320,26 @@ export class GrpcPlugin extends BasePlugin<grpc> {
     const plugin = this;
     return (original: typeof grpcTypes.makeGenericClientConstructor): never => {
       plugin._logger.debug('patching client');
-      return function makeClientConstructor<ImplementationType>(
+      return function makeClientConstructor(
         this: typeof grpcTypes.Client,
-        methods: grpcTypes.ServiceDefinition<ImplementationType>,
+        methods: { [key: string]: { originalName?: string } },
         serviceName: string,
         options: grpcTypes.GenericClientOptions
       ) {
         // tslint:disable-next-line:no-any
         const client = original.apply(this, arguments as any);
+        const methodsToWrap = [
+          ...Object.keys(methods),
+          ...(Object.keys(methods)
```

Comment:
This one probably deserves its own function call so it is clear why you are doing this.

I would create a function named something like `getMethodsToWrap` which in turn calls something like `getOriginalMethodNames`.